common/app/views/support/package.scala

```diff
@@ -184,7 +184,7 @@ object RenderOtherStatus {
   def apply(result: Result)(implicit request: RequestHeader, context: ApplicationContext) = result.header.status match {
     case 404 => NoCache(NotFound)
     case 410 if request.isJson => Cached(60)(JsonComponent(gonePage, "status" -> "GONE"))
```

Comment:
I'm not entirely clear on what implications that might have for the `json` use case either.  I think it might be worth investigating because, like you say, it would be nice if this were consistent.  My feeling is that should be in another PR and is not a priority.